src/libraries/icubmod/parametricCalibratorEth/parametricCalibratorEth.cpp

```diff
@@ -688,35 +697,39 @@ bool parametricCalibratorEth::calibrate()
         }
 
         //4) check calibration result
-        if(checkCalibrateJointEnded((*Bit)) ) //check calibration on entire set
+        std::list<int> failedJoints = {};
+        if(checkCalibrateJointEnded(*Bit, failedJoints )) //check calibration on entire set
         {
-            yDebug() <<  deviceName  << ": set" << setOfJoint_idx  << ": Calibration ended, going to zero!\n";
+            yDebug() << deviceName  << ": set" << setOfJoint_idx  << ": Calibration ended, going to zero!";
         }
         else    // keep pid safe  and go on
         {
-            yError() <<  deviceName  << ": set" << setOfJoint_idx << ": Calibration went wrong! Disabling axes and keeping safe pid limit\n";
-
+            yError() <<  deviceName  << ": set" << setOfJoint_idx << ": Detected errors during calibration! Idling failed joints; Remaining joints set with safe PWM limits";
```

Comment:
Hi @MSECode, is this error message correct? The remaining joint will have the original value of pwm limit, haven't they?